packages/model-viewer/src/three-components/SmoothControls.ts

```diff
@@ -166,18 +165,13 @@ export class SmoothControls extends EventDispatcher {
   enableInteraction() {
     if (this._interactionEnabled === false) {
       const {element} = this;
-      element.addEventListener('mousemove', this.onPointerMove);
-      element.addEventListener('mousedown', this.onPointerDown);
+      element.addEventListener('mousedown', this.onMouseDown);
       if (!this._disableZoom) {
         element.addEventListener('wheel', this.onWheel);
       }
       element.addEventListener('keydown', this.onKeyDown);
       element.addEventListener(
-          'touchstart', this.onPointerDown, {passive: true});
-      element.addEventListener('touchmove', this.onPointerMove);
-
-      self.addEventListener('mouseup', this.onPointerUp);
-      self.addEventListener('touchend', this.onPointerUp);
+          'touchstart', this.onTouchStart, {passive: false});
```

Comment:
It cannot be `{passive: true}` otherwise I am able to reproduce intermittent behavior where the `e.preventDefault()` to prevent scroll in the `touchmove` handler is not respected. I suspect it's because the attachment of the `touchmove` event listener (which has `{passive: false}`) happens in this `touchstart` event listener (which in turn must be non-passive too?).

We could omit the `{passive: false}` altogether because even with [Chrome's passive event listener intervention](https://developers.google.com/web/updates/2017/01/scrolling-intervention#the_intervention), `passive` for this listener will default to `false` **but** if we do, Chrome will print a `[Violation] Added non-passive event listener to a scroll-blocking...` warning in the console.

Same goes for the `touchmove` listener. For now we can remove `{passive: false}` since it's the default but once we attach it to window as per #2682, we will definitely have to explicitly declare it again. (Again, we should keep it just to avoid the violation warning)

Right now Chrome complains about the `non-passive` event listener on `wheel` too. Same as above, we can fix being explicitly adding `{passive: false}`

Reply:
Thanks for the explanation, sounds good. Let's fix the wheel one as well in a follow-on.